### DISCUSSION TITLE
[Table Panel for Tempo] Only want to see searched span row , do not want to see trace id row.

### DISCUSSION THREAD

**Discussion by pingping95** [grafana/grafana#73135] [Dashboards, Panels & Visualizations]
Related : https://grafana.slack.com/archives/C01D981PEE5/p1691556832410299

<img width="1232" alt="image" src="https://github.com/grafana/grafana/assets/67780144/8921b47a-c422-45b3-a8b5-3a8d551f1f96">

I`m creating Graph Panel from Tempo datasource.

I have a question.

Our company `starts Trace from API Gateway` and spreads it to Backend.

In this case, if I search Trace through the Grafana Table view, the Root Trace ID will be visible first.

but it was a little uncomfortable because `**only api gateway span name was shown first in the table view**`.

In fact, most developers are interested in their services, particular Spans, not Root Spans.

```
{resource.service.name="$service" && name="$span" && span.http.status_code=$status_code} | select(span.http.target, span.http.method)
```

This is my graph panel tempo query.

If i want to see http.target and http.method , **_I should have to click button all traces_**. It`s somewhat hard.

Please could improve this inconvenience?

Or is there a way to make Trace id row invisible on the Graph panel?

Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82550. Feel free to continue the discussion around this there. Thank you!
